src/Microsoft.IdentityModel.Tokens/Validation/ValidationFailureType.cs

```diff
@@ -134,5 +134,10 @@ private class XmlValidationFailure : ValidationFailureType { internal XmlValidat
         /// </summary>
         public static readonly ValidationFailureType IssuerValidatorThrew = new IssuerValidatorFailure("IssuerValidatorThrew");
         private class IssuerValidatorFailure : ValidationFailureType { internal IssuerValidatorFailure(string name) : base(name) { } }
+
+        /// <summary>
+        /// Defines a type that represents that the token type validation delegate threw and exception.
```

Comment:
```suggestion
        /// Defines a type that represents the fact that the token type validation delegate threw an exception.
```
Small suggestion for more clarity

Reply:
Thanks. Sounds better this way.